app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
Hmm yes I see there's a mismatch with the whole `/someobject/<object_id>/someaction` thing manipulating more than one object - it's a pseudo-object-oriented interface and it's like you're addressing something not in context of its collection.